runtime/integration-tests/src/evm/precompile.rs

```diff
@@ -58,6 +58,10 @@ async fn axelar_precompile_execute() {
 
 	let currency_id = CurrencyId::ForeignAsset(123456);
 
+	// Address number comes from:
```

Comment:
For clarity, I think it's the other way around. Our solidity contract is using the address defined here, on chain.

Reply:
Ok, thanks both for the clarification!